inheritance/src/test/java/inheritance/RestaurantTest.java

```diff
@@ -5,32 +5,34 @@
 import static org.junit.Assert.assertEquals;
 
 public class RestaurantTest {
+    Restaurant restaurant = new Restaurant("SteakHouse", 3, 2);
 
```

Comment:
Using a before method can handy for tests. It will run every time before each test, and be helpful to reset the tests each time so they are consistent.
`@Before
public void setup(){
    restautant = new Restaurant("SteakHouse", 3, 2);
}`
source; https://www.baeldung.com/junit-before-beforeclass-beforeeach-beforeall